src/parser/jobs/blm/RotationWatchdog.js

```diff
@@ -193,7 +202,7 @@ export default class FireCounter extends Module {
 					key: 'title-' + rotation.start,
 					content: <Fragment>
 						{this.parser.formatTimestamp(rotation.start)}
-						&nbsp;-&nbsp;{this._renderCount(rotation.fire4Count, rotation.missingCount)} Fire IVs
+						&nbsp;-&nbsp;{this._renderCount(rotation.fire4Count, rotation.missingCount.missing)} / {rotation.missingCount.expected} Fire IVs
```

Comment:
Try to avoid using `&nbsp;` since they tend to make code a bit unreadable.
You could use `<span> - </span>` to the same effect.
It also helps grouping your lines in that into different span groups, which might help later down the line with styling them more easily.

Reply:
I'm guilty as charged on this :blobsweat:.

I've seen some other projects use `{' - '}` to achieve what I assume this does - thoughts on that approach?

Reply:
Also works fine, but has the downside that it requires both sides of that expression to have a string that's not empty to work

Reply:
everything addded now